src/sentry/static/sentry/app/views/performance/vitalsCards.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import Card from 'app/components/card';
+import Link from 'app/components/links/link';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+import EventView from 'app/utils/discover/eventView';
+import {getAggregateAlias, WebVital} from 'app/utils/discover/fields';
+import {formatPercentage} from 'app/utils/formatters';
+import VitalsCardsDiscoverQuery from 'app/views/performance/vitalDetail/vitalsCardsDiscoverQuery';
+
+import {
+  vitalAbbreviations,
+  vitalDetailRouteWithQuery,
+  vitalsBaseFields,
+  vitalsThresholdFields,
+} from './vitalDetail/utils';
+
+type Props = {
+  eventView: EventView;
+  organization: Organization;
+  location: Location;
+};
+
+export default function VitalsCards(props: Props) {
+  const {eventView, organization, location} = props;
+  const vitalsView = eventView.clone();
+
+  const shownVitals = [
+    WebVital.FP,
+    WebVital.FCP,
+    WebVital.LCP,
+    WebVital.FID,
+    WebVital.CLS,
+  ];
+
+  return (
+    <VitalsCardsDiscoverQuery
+      eventView={vitalsView}
+      orgSlug={organization.slug}
+      location={location}
+    >
+      {({isLoading, tableData}) => (
+        <VitalsContainer>
+          {shownVitals.map(vitalName => (
+            <LinkedVitalsCard
+              key={vitalName}
+              vitalName={vitalName}
+              tableData={tableData}
+              isLoading={isLoading}
+              {...props}
+            />
+          ))}
+        </VitalsContainer>
+      )}
+    </VitalsCardsDiscoverQuery>
+  );
+}
+
+const VitalsContainer = styled('div')`
+  display: grid;
+  grid-auto-flow: column;
+  gap: ${space(2)};
+  margin-bottom: ${space(2)};
+`;
+
+type CardProps = Props & {
+  vitalName: WebVital;
+  tableData: any;
+  isLoading?: boolean;
+  noBorder?: boolean;
+};
+
+const NonPanel = styled('div')`
+  flex-grow: 1;
+`;
+
+const StyledVitalCard = styled(Card)`
+  color: ${p => p.theme.textColor};
+  padding: ${space(1.5)} ${space(2)} ${space(2)} ${space(2)};
+
+  &:focus,
+  &:hover {
+    color: ${p => p.theme.textColor};
+    top: -1px;
+  }
+`;
+
+export function LinkedVitalsCard(props: CardProps) {
+  const {vitalName} = props;
+  return (
+    <VitalLink {...props} vitalName={vitalName}>
+      <VitalsCard {...props} />
+    </VitalLink>
+  );
+}
+
+export function VitalsCard(props: CardProps) {
+  const {isLoading, tableData, vitalName, noBorder} = props;
+
+  const measurement = vitalAbbreviations[vitalName];
+
+  const Container = noBorder ? NonPanel : StyledVitalCard;
+
+  if (isLoading || !tableData || !tableData.data || !tableData.data[0]) {
+    return (
+      <Container interactive>
+        <CardTitle>{t(`${measurement} Passing`)}</CardTitle>
+        <CardValue>-</CardValue>
```

Comment:
We use this em dash in various places because it visually looks better 💅 

```suggestion
        <CardValue>{'\u2014'}</CardValue>
```